docs/api/webxrHardwareExtensionAPIs.md

```diff
@@ -6,241 +6,35 @@ order: 2
+  session.addEventListener('meshremove', _meshremove);
+  session.addEventListener('planeadd', _planeadd);
+  session.addEventListener('planeremove', _planeremove);
 ```
 
-#### `MLHandUpdate.gesture : String`
-
-The current gesture pose of the hand. One of:
-
-- `null` (no gesture detected)
-- `'finger'`
-- `'fist'`
-- `'pinch'`
-- `'thumb'`
-- `'l'`
-- `'openHandBack'`
-- `'ok'`
-- `'c'`
-
-##### Notes:
-
-- A bone may be `null`, which means it is not currently detected.
-- The thumb has one less bone than the other fingers.
-
-### `MLTransform`
-
-A generic container for position/rotation data in world space.
-
-#### `MLTransform.position : Float32Array(3)`
-
-A three-component world position vector.
-
-#### `MLTransform.rotation : Float32Array(4)`
-
-A four-component world quaternion.
-
-### `MLEyeTracker`
-
-Used to acquire eye tracking updates from the Magic Leap platform.
-
-#### `MLEyeTracker.oneye : function(MLEye[2])`
-
-When set, `oneye` will be called with an array of `MLEyeUpdate`. This indicates an update to the user's detected eye post by the Magic Leap platform.
-
-Both eyes are present in all updates.
-
-### `MLEye`
-
-A single eye state as detected by the platform.
-
-### `MLEye.position : Float32Array(3)`
-
-The world position of the eye origin as a vector.
-
-> Do not use this for checking where the eye is looking; that is what `fixation` is for.
-
-### `MLEye.rotation : Float32Array(4)`
-
-The rotation of the eye origin as a world quaternion.
-
-> Do not use this for checking where the eye is looking; that is what `fixation` is for.
-
-### `MLEye.blink : Boolean`
-
-Whether this eye is currently closed (`true`) or open (`false`).
-
-### Endpoints
-
-#### `browser.magicleap.RequestMeshing() : MLMesher`
-
-Returns an instance of `MLMesher`, which can be used to receive world meshing buffer updates from the Magic Leap platform.
-
-#### `browser.magicleap.RequestPlaneTracking() : MLPlaneTracker`
-
-Returns an instance of `MLPlaneTracker`, which can be used to receive world planes detected by the Magic Leap platform.
-
-#### `browser.magicleap.RequestHandTracking() : MLHandTracker`
-
-Returns an instance of `MLHandTracker`, which can be used to receive hand tracking data from the Magic Leap platform.
-
-#### `browser.magicleap.RequestEyeTracking() : MLEyeTracker`
-
-Returns an instance of `MLEyeTracker`, which can be used to receive eye tracking data from the Magic Leap platform.
-
-#### `browser.magicleap.RequestDepthPopulation(populateDepth : Boolean)`
-
-Sets whether the render loop will populate the depth buffer by using the meshing subsystem.
+Each extension behaves differently:
+- When the meshing mechanism for an `InputSource` is invoked, update events for `meshadd`, `meshupdate`, and `meshremove` are fired on `session` (the `XRSession`).
+- Planes behave similarly as meshing as there are two update events emitted: `planeadd' and 'planeremove' which are fired on `session` (the `XRSession`).
```

Comment:
Weird ticks on `planeadd`, `planeremove`.

Reply:
Do you mean they shouldn't be ticked here?

Reply:
I mean the tick syntax looks incorrect.

```
`planeadd'
```

Reply:
Oh I see, thanks. I'll fix and merge.